lib/versioned/^4.0.0/index.js

```diff
@@ -48,8 +48,8 @@ function execute(opts, env) {
       tree = gulpInst.tree({ deep: true });
       tree.label = 'Tasks for ' + chalk.magenta(tildify(env.configPath));
 
-      return logTasks(tree, function(task) {
-        return gulpInst.task(task).description;
+      return logTasks(tree, opts.depth, function(taskname) {
+        return gulpInst.task(String(taskname.toString()));
```

Comment:
I removed these and everything still seems to work.  Just trying to figure out if you encountered something I haven't